src/meta/api/src/schema_api_impl.rs

```diff
@@ -1929,14 +2200,44 @@ impl<KV: KVApi> SchemaApi for KV {
     }
 }
 
+async fn remove_table_stage_files(
+    kv_api: &impl KVApi,
+    tenant: &str,
+    db_id: u64,
+    table_id: u64,
+    condition: &mut Vec<TxnCondition>,
+    if_then: &mut Vec<TxnOp>,
+) -> Result<(), MetaError> {
+    // List files by tenant, db_name, table_name
+    let dbid_tbname_idlist = TableStageFileNameIdent {
+        tenant: tenant.to_owned(),
+        db_id,
+        table_id,
+        // Using a empty file to to list all
+        file: "".to_string(),
+    };
+
+    let files = list_keys(kv_api, &dbid_tbname_idlist).await?;
+    for file in files {
+        let (file_seq, _opt): (_, Option<TableStageFileInfo>) =
+            get_struct_value(kv_api, &file).await?;
+        if file_seq != 0 {
+            condition.push(txn_cond_seq(&file, Eq, file_seq));
+            if_then.push(txn_op_del(&file));
+        }
+    }
+
+    Ok(())
+}
```

Comment:
Inconsistent write:

The file list may change after `list_keys` at line-2220.

To assert consistency, every time copying-files change, the seq of table-meta must increase. So that when deleting these files, by checking table-meta-seq it guarantees there are not changes to the file list.

This is similar to the db-table pattern: a **parent** node is responsible for the consistency of the **set** of its child nodes.